### DISCUSSION TITLE
TypeError on production

### DISCUSSION THREAD

**Discussion by anch09** [atomiks/tippyjs#1132] [Q&A]
Hello,

I hope here I can find an answer to my issue, I've been using Tippy react since a while ago but today I pushed an update to my project but for my surprise it shows an error message in the console saying e.fn does not exists:

<img width="1510" alt="Screenshot 2023-07-19 at 1 31 21 PM" src="https://github.com/atomiks/tippyjs/assets/77537495/132b6fc7-70d5-4699-81be-33b38b78fd3b">

which I've been looking into the library and it corresponds to the file `tippy-bundle.umd.js` at the function `createTippy` lines 695 to 697: 

<img width="597" alt="Screenshot 2023-07-19 at 1 32 54 PM" src="https://github.com/atomiks/tippyjs/assets/77537495/836b9185-c62a-4417-a042-46655779a6d8">

I'm not sure why is this happening while I haven't change any tooltip logic. It only happens on production, when is build but not on development.

I'm using nextjs 13 tho.

Thanks in advance for any help you could provide.

**Comment by kdawgwilk** [comment]
Looks like its related to new versions of NextJS due to a bug in SWC https://github.com/atomiks/tippyjs-react/issues/379
